x/validator-preference/types/params.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+var _ paramtypes.ParamSet = (*Params)(nil)
```

Comment:
just curious, why not ? I've been trying to search for differences amongst these two
```suggestion
_ paramtypes.ParamSet = &Params{}
```

Reply:
unfortunately that doesnot work 
```
syntax error: non-declaration statement outside function body
```